spec/services/schools/teacher_email_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe Schools::TeacherEmail do
+  let(:email) { "test@example.com" }
+  let(:trn) { "123456" }
+  subject { described_class.new(email:, trn:) }
+
+  describe "#is_currently_used?" do
```

Comment:
Does this service spec give us sufficient confidence? It is just testing the mocked values without looking at active periods and teachers.

Reply:
@peterdavidhamilton  I've pushed a commit changing them to integration tests.